lib/get-tasks.spec.js

```diff
@@ -2,51 +2,44 @@ const getTasks = require('./get-tasks.js');
 
 const correctSources = [
   [
-    "This PR closes #BUNNNY-1",
+    "This PR closes #BUNNY-1",
     [
-      { taskId: "BUNNNY-1", isCustom: true }
+      { taskId: "BUNNY-1" }
     ]
   ],
   [
-    "This PR closes #86bwmzcyf",
+    "This PR has duplicated reference #BUNNY-1 #BUNNY-1",
     [
-      { taskId: "86bwmzcyf", isCustom: false }
+      { taskId: "BUNNY-1" },
     ]
   ],
   [
-    "This PR closes #86bwmzcyf but it's in the middle of the text",
+    "This PR closes #CORE-123 and is related to #BUNNY-1",
```

Comment:
are these tests relevant when we only check PR title, not description? The title has a structured format (see e.g. my regex101 screenshot in prev comment)

Reply:
Matching function tested here does not have any assumptions regarding the input format, thus I don't think we really need to adjust test cases to match the real use case. We can just focus on testing its functionality, that is detecting issue reference at the end of the input.

Reply:
yeah but if we change the regex as I'm proposing then it will be invalid :D

Reply:
True, I've already adjusted tests 10a4277c943f3cd19d7742b5eabc76420e70ae35